index.html

```diff
@@ -671,11 +671,12 @@
   - **`git tag`**
 - Checkout to a specific tag
   - **`git checkout <tag_name>`**
+  - this puts you in a detached HEAD state
 - Pushing tags to the remote
   - **`git push <push_options_and_branches> --tags`**
 - Delete tag
   - **`git tag -d <tag_name>`**
-- Change where a tag points
+- Change the commit pointed at by a tag
```

Comment:
maybe "Change the commit a tag points to" is better?

Reply:
OK